packages/terra-consumer-icon/src/cerner-one-icons.csv

```diff
@@ -0,0 +1,154 @@
+add,src/icons/0001_plus_a.svg,themeable ,
```

Comment:
this will be deleted and consumer-icon-mapping.csv will be added once we have mapping file.